vagrant-spk

```diff
@@ -271,6 +276,21 @@ def auto(args):
     print('$', 'vagrant-spk init')
     init(args)
 
+    # If this is a Meteor app, modify the app:
+    #
+    # - Add the Sandstorm accounts package, for auto-sign-in.
+    #
+    # - Maybe in the future, fix HTTP/HTTPS mixed content stuff on
+    #   common CDN domains. Note that the real fix for this is to
+    #   embed a copy of the assets in the app package, but this works
+    #   for now.
+    if stack == STACK_TYPE_METEOR:
+        ACCOUNTS_PKG = 'kenton:accounts-sandstorm'
+        if ACCOUNTS_PKG not in open('.meteor/packages').read():
+            with open('.meteor/packages', 'w') as fd:
+                fd.write(ACCOUNTS_PKG)
```

Comment:
I would add `fd.write('\n')` before this line to ensure the package starts on a newline.